benchmark/Cargo.toml

```diff
@@ -18,3 +18,4 @@ path = "src/rwlock.rs"
 
 [features]
 nightly = ["parking_lot/nightly"]
+deadlock_detection = ["parking_lot/deadlock_detection"]
```

Comment:
The benchmarks don't need deadlock detection.

Reply:
It's not enabled by default, it's useful to measure the impact.

Reply:
Fair enough